recipe/conda_build_config.yaml

```diff
@@ -50,8 +50,8 @@ m2w64_fortran_compiler:        # [win]
 CMAKE_GENERATOR:               # [win]
   - NMake Makefiles            # [win]
 
-cuda_compiler:                 # [linux64 or win]
-  - nvcc                       # [linux64 or win]
+cuda_compiler:                 # [linux64 or win or aarch64 or ppc64le]
+  - nvcc                       # [linux64 or win or aarch64 or ppc64le]
```

Comment:
```suggestion
cuda_compiler:                 # [linux or win]
  - nvcc                       # [linux or win]
```

Reply:
Thx. I thought about this but was worried it's not general enough.

Reply:
Yeah `linux` covers all 3 architectures